src/core/ext/filters/client_channel/lb_policy.h

```diff
@@ -95,7 +95,9 @@ class LoadBalancingPolicy
   /// Updates the policy with a new set of \a args from the resolver.
   /// Note that the LB policy gets the set of addresses from the
   /// GRPC_ARG_LB_ADDRESSES channel arg.
-  virtual void UpdateLocked(const grpc_channel_args& args) GRPC_ABSTRACT;
+  /// Returns true if the update caused the number of backends to go from
```

Comment:
There's no need to push this functionality into the LB policy API.  The client channel can directly look at the addresses returned by the resolver (in fact, we're already doing so in one case; see line 362 in client_channel.cc), so let's do that instead of broadening the LB policy API.

Reply:
Done

Reply:
Actually, I do need to access the old num addresses. Is there a way to accomplish that without adding to LB API?

Reply:
done